src/modules/endpoint/__tests__/Endpoint.spec.js

```diff
@@ -85,22 +85,26 @@ describe('Endpoint', () => {
   describe('Actions', () => {
     let clock
     const mockedTime = 0
+    let prevError
 
     beforeEach(() => {
       clock = lolex.install({now: mockedTime, toFake: []})
+      prevError = console.error
+      console.error = error => console.log(`Some expected error was thrown: ${error}`)
```

Comment:
(Why) Do we need a console.log ?

Reply:
We want to let the tests fail if console.error is executed. So we need to overwrite console.error before doing something which is supposed to output an error so the test does not fail.

Reply:
Yep, thats clear. Why did you choose to overwrite with a console.log? Why not a noop? For better debugging?